include/mbedtls/platform.h

```diff
@@ -116,12 +116,13 @@ extern void * (*mbedtls_calloc)( size_t n, size_t size );
 extern void (*mbedtls_free)( void *ptr );
 
 /**
- * \brief   Set your own memory implementation function pointers
+ * \brief   This function sets your own memory implementation function 
+ *          pointers.
  *
- * \param calloc_func   the calloc function implementation
- * \param free_func     the free function implementation
+ * \param calloc_func   The calloc function implementation.
```

Comment:
I'm not sure, but is that something where you'd want to use `\c` for the function names, too?

Reply:
You are correct! Thanks for catching. Fixed on all other than see read and write as these did not seem like the actual function names.